bindings.h

```diff
@@ -0,0 +1,2 @@
+#include "ds18b20.h"
+#include "onewire_bus.h"
```

Comment:
Two points:
- if we introduce the usage of the onewire_bus.h in our api than we can directly include them in esp-idf-sys, and thous not needing the extra binding.h. That will than be conditionally included if the onewire component is included. I will update the changes for you in esp-idf-sys. 
- Lets focus in this PR on the onewire-bus itself first, so only parts of it are included and not subsequent other stuff.

Reply:
I will hide the onewire_bus behind an `#ifdef`. We can remove the ds18b20 for now.

I welcome you to update the `bindings.h` in esp-idf-sys, or I can make a related PR.

Reply:
I struggled to get this to work, with 
```
#ifdef ESP_IDF_COMP_ESPRESSIF__ONEWIRE_BUS_ENABLED
#include "onewire_bus.h"
#include "onewire_device.h"
#endif 
```

The `onewire` module in `bindings.rs` would be empty. Any ideas?

Reply:
I included it in esp-idf-sys for now, can you check if that is working for you. The module itself is optional. It did work for you without specifying the module before or did you always used the module?

Reply:
Done. It previously would only populate code inside the module because of the `Cargo.toml` `bindings_module` option. But inside `bindings.h`, the ESP_IDF_COMP_ESPRESSIF__ONEWIRE_BUS_ENABLED would always evaluate to 0 in my local setup. It works now with `bindings.h` in the `esp-idf-sys` crate, but now we're missing `onewire_device.h`.